backend/app/models/users.js

```diff
@@ -39,6 +39,7 @@ var columns = [
     'affiliation',
     'isAnonymous',
     'langId',
+    'authId',
```

Comment:
Why are we adding the `authId` to the User's table. Aren't we already using the email to associate an Indaba Users table row with an auth service user?

Reply:
The issue is that when we store the review comments in survey service, they're being stored with the auth service's userIds, which are different than those of Greyscale. The survey service's review comments don't return the user's email (and doesn't know what they are), so we need them.